packages/cells/src/widget.ts

```diff
@@ -238,13 +238,7 @@ export class Cell<T extends ICellModel = ICellModel> extends Widget {
 
     // Editor settings
     if (options.editorConfig) {
-      let editorOptions: any = {};
-      Object.keys(options.editorConfig).forEach(
-        (key: keyof CodeEditor.IConfig) => {
-          editorOptions[key] = options.editorConfig?.[key] ?? null;
-        }
-      );
-      this.editor.setOptions(editorOptions);
+      this.editor.setOptions({...options.editorConfig} ?? {});
```

Comment:
Looks like this could be simplified to:

```suggestion
      this.editor.setOptions({...options.editorConfig});
```

(caught by the CodeQL check)